packages/rx_bloc_cli/mason_templates/bricks/rx_bloc_base/__brick__/docs/continuous_delivery.md

```diff
@@ -0,0 +1,106 @@
+## Fastlane
```

Comment:
Make sure this file/folder is copied to the example folder the same way as the the main README.file is copied